packages/upload-api/src/utils/did-mailto.js

```diff
@@ -0,0 +1,10 @@
+import * as DidMailto from '@web3-storage/did-mailto'
+
+/**
+ * @param {import("@web3-storage/did-mailto/dist/src/types").DidMailto} mailtoDid
+ */
+export function emailAndDomainFromMailtoDid(mailtoDid) {
```

Comment:
Nit: Feels like it should be two different functions `toEmail(did): Email` and `toDomain(email: Email): string`

Reply:
done!